tools/cldr-apps/js/test/nonbrowser/test-cldrXpathUtils.mjs

```diff
@@ -1,7 +1,7 @@
-const expect = require("chai").expect;
-const mocha = require("mocha");
+import { expect } from "chai";
+import mocha from "mocha";
```

Comment:
so this file isn't just renamed, it's converted from CJS (with js extension) to MJS (with mjs extension)

and that's good -- the more we can use MJS instead of CJS the better